run

```diff
@@ -270,6 +292,28 @@ def build_swift_toolchain(workspace, args):
         raise common.UnsupportedPlatform
     common.check_execute(build_command, timeout=9999999)
 
+def build_clang(workspace, source_path):
+    build_path = os.path.join(workspace, 'build_clang_source_compat')
+    source_path = os.path.join(source_path, 'llvm')
+    common.check_execute(['mkdir', '-p', build_path])
+
+    with common.DirectoryContext(build_path):
+        try:
+            ninja_path = common.check_execute_output(['xcrun', '--find', 'ninja']).strip().decode("utf-8")
+            cmake_command = [
+                'xcrun', 'cmake', '-G', 'Ninja',
+                '-DCMAKE_MAKE_PROGRAM={}'.format(ninja_path),
+                '-DLLVM_ENABLE_PROJECTS=clang;llvm',
+                '-DCMAKE_BUILD_TYPE=Release',
+                '-DCLANG_APPLE_BUILD_VERSION_STRING=13000000',
+                '-DLLVM_TARGETS_TO_BUILD=X86;AArch64;ARM',
+                source_path]
+            common.check_execute(cmake_command)
+            common.check_execute(['xcrun', 'ninja'])
+        except:
+            raise
```

Comment:
Why is the `try catch` block needed if you are just raising the exception?

Reply:
Removed it. Thanks!